extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -400,30 +400,51 @@ class UpdateReplicationStatus extends BackbeatTask {
         return null;
     }
 
+    _updateStatusDurationMetric(sourceEntry, err, replicationStatus) {
+        const result = (() => {
+            if (err) {
+                return 'error';
+            }
+            if (replicationStatus) {
+                return 'success';
+            }
+            return 'nochange';
+        })();
```

Comment:
I don't feel like a function here is any better then two if statements.
```suggestion
        let result = 'nochange';
        if (err) {
            result = 'error';
        }
        else if (replicationStatus) {
            result = 'success';
        }
```